psi4/driver/json_wrapper.py

```diff
@@ -123,7 +124,10 @@ def run_json_qc_schema(json_data, clean):
     _clean_psi_environ(clean)
 
     # This is currently a forced override
-    if json_data["schema_version"] != 0:
+    if json_data["schema_name"] != "qc_schema_input":
+        raise KeyError("Schema name of '{}' not understood".format(json_data["schema_name"]))
+
+    if json_data["schema_version"] != 1:
```

Comment:
plan is to add to if/else here (and any other place schema processing forks) when want to run e.g., `1.dev5` through psi?

Reply:
sorry, `2.dev5`